04_UpdateUser.php

```diff
@@ -2,106 +2,104 @@
 <html lang = "eng">
```

Comment:
nothing new other than change in positioning and same colours + fonts as the other admin pages